csfieldguide/templates/interactives/data-visualisation.html

```diff
@@ -0,0 +1,67 @@
+{% extends interactive_mode_template %}
+
+{% load i18n %}
+{% load static %}
+
+{% block html %}
+  <div class="container-fluid my-3 text-center">
+    <div id="data-vis-title" class="col-12">{% trans 'Data Visualisation' %}</div>
+    <div id="data-vis-instructions" class="col-12">
+      <div class="col-12">
+        {% blocktrans %}
+        Find the mode (most common number) of the data below.
+        You will have a limited amount of time to view the data.
+        {% endblocktrans %}
+      </div>
+      <button id="button-start" type="button" class="btn btn-success my-1">{% trans 'Start' %}</button>
+    </div>
+    <div id="data-vis-guesser" class="col-12 d-none">
+      <label for="data-vis-selector">{% trans 'What was the mode (most common number) of the data?' %}</label>
+      <form class="form-inline justify-content-center">
+        <select id="data-vis-selector" class="form-control text-center d-inline mx-2" size="1">
+          <option>0</option>
+          <option>1</option>
+          <option>2</option>
+          <option>3</option>
+          <option>4</option>
+          <option>5</option>
+          <option>6</option>
+          <option>7</option>
+          <option>8</option>
+          <option>9</option>
+        </select>
+        <button id="button-submit" type="button" class="btn btn-secondary d-inline mx-2">{% trans 'Submit' %}</button>
```

Comment:
Add padding for smaller screens

![image](https://user-images.githubusercontent.com/16066822/68162965-3172ff80-ffbe-11e9-95b6-26d5e461bf2d.png)